components/base/sensorcontrolled/sensorcontrolled.go

```diff
@@ -151,6 +159,38 @@ func (sb *sensorBase) Reconfigure(ctx context.Context, deps resource.Dependencie
 		return errors.Wrapf(err, "no base named (%s)", newConf.Base)
 	}
 
+	if sb.velocities != nil {
+		sb.lvctrl = &linearVelControllable{
```

Comment:
i think you should have one loop if both controls the same thing

Reply:
They don't control the same thing - one is angular, one is linear.

Reply:
~~But the loops have exactly the same inputs and (including the control blocks). Am I missing something?~~

I see that the base wrappers are different, never mind